namespaces/docker.yml

```diff
@@ -9,3 +9,103 @@ namespace:
     type: string
     description: >
       Docker container id
+  - name: event
+    type: group
+    description: >
+      Event emitted by docker daemon that describes some activity that occurred in the past
+    fields:
+    - name: action
+      type: string
+      example: create
+      description: >
+        What activity occurred
+    - name: status
+      type: string
+      example: create
+      description: >
+        What is the result of the activity
+    - name: type
+      type: string
+      example: container
+      description: >
+        Container or image or volume or network or daemon or plugin or service or node or secret or config
```

Comment:
Note that there isn't an `enum` type: https://www.elastic.co/guide/en/elasticsearch/reference/2.4/mapping-types.html

but you could document the available choices, something like this: https://github.com/ViaQ/elasticsearch-templates/blob/master/namespaces/_default_.yml#L63